### ISSUE TITLE
[WMS or WFS mode] Add a workspace option

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/sdi-consistence-check#43]
It would be appreciate to have the possibility to check only on a workspace.

But question : would it be specific to GeoServer ?
But it would be an option...

**Comment by fvanderbiest** [comment]
Checking `/geoserver/ws/ows` does the job, I do not see the use ...

**Comment by MaelREBOUX** [comment]
Yes, but I can't modify the URL in Jenkins. If it is an option, we could modify the jenkins. No ?
